backend/src/utils/error.ts

```diff
@@ -0,0 +1,49 @@
+import httpStatus from "http-status";
+
+class CustomError {
```

Comment:
why are you not extending this from the error instance

Reply:
I implemented the CustomError so that we can handle some errors and display the response to the user. Any error not handled here is a runtime error and doesn't need to be displayed to user and can be logged.